style.css

```diff
@@ -0,0 +1,473 @@
+.body {
+    max-width: 1920px;
+    margin: 0 auto;
+    background: url('pic/iswanto-arif-448003-unsplash.png') no-repeat;
+    background-size: cover;
+    overflow: visible;
+    height: 100vh;
+}
+.header {
+    width: 100%;
+    padding: 87px 114px 0 199px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    margin: 0 0 130px 0;
+}
+.header:after{
+    content: "";
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    background-color: #fff;
+    top: -100vh;
+    left: 0;
+    z-index: 2;
+    transition: .3s ease;
+}
+header > img{
+    margin-top: 1px;
+}
+.container {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+}
+.nav {
+    display: flex;
+    justify-content: center;
+    margin-right: 4px;
+    letter-spacing: .25pt;
+}
+.nav__link:not(:last-child) {
+    margin: 0 35px 0 0;
+}
+.nav__link {
+    color: #fff;
+    text-decoration: none;
+    font-size: 22px;
+    line-height: 26px;
+    font-family: 'Museo';
+    border-bottom: 1px solid transparent;
+    transition: .3s ease;
+}
+.nav__link:hover {
+    border-bottom: 1px solid #ffffff;
+    opacity: 0.8;
+}
+.nav__toggle {
+    display: none;
+}
+.nav__burder{
+    display: none;
+}
+.bali {
+    display: flex;
+    flex-direction: column;
+    margin-left: 60px;
+    padding-left: calc(100vw - 1780px)
+}
+.bali-wrapper {
+    position: relative;
+
+}
+.bali__title {
+    display: flex;
+    font-family: 'PTSerif';
+    font-style: normal;
+    margin: 0;
+    font-weight: bold;
+    font-size: 280px;
+    color: #f8eee4;
+    opacity: 0.4;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)